_includes/posts/recent_news.html

```diff
@@ -0,0 +1,11 @@
+<div class="featured-posts">
+  {% for post in site.posts limit: 4 %}
+  <div class="post-teaser">
+    <a href="{{ post.url }}" class="post-title">{{ post.title }}</a>
+
+    {% include posts/meta.html post=post %}
+
+    <div class="excerpt">{{ post.summary | default: post.excerpt | markdownify }}</div>
```

Comment:
I was expecting all this to be just an include, given that it's the same as in the post page. Can be done in a follow up, of course.

Reply:
Yeah, that makes a lot of sense.